source/Mission.cpp

```diff
@@ -1033,34 +1068,59 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 	if(!result.stopovers.empty())
 	{
 		string planets;
+		vector<const Government *> stopoverGovernments;
 		const Planet * const *last = &*--result.stopovers.end();
 		int count = 0;
 		for(const Planet * const &planet : result.stopovers)
 		{
 			if(count++)
 				planets += (&planet != last) ? ", " : (count > 2 ? ", and " : " and ");
 			planets += planet->Name() + " in the " + planet->GetSystem()->Name() + " system";
+			stopoverGovernments.emplace_back(planet->GetGovernment());
 		}
 		subs["<stopovers>"] = planets;
+		
+		// Stopover governments: "<government name>" with "," and "and", e.g.
+		// "Free Worlds and Republic", "Free Worlds, Republic, and Syndicate".
+		subs["<government: stopovers>"] = GovernmentNames(stopoverGovernments);
 	}
 	// Waypoints: "<system name>" with "," and "and".
 	if(!result.waypoints.empty())
 	{
 		string systems;
+		vector<const Government *> waypointGovernments;
 		const System * const *last = &*--result.waypoints.end();
 		int count = 0;
 		for(const System * const &system : result.waypoints)
 		{
 			if(count++)
 				systems += (&system != last) ? ", " : (count > 2 ? ", and " : " and ");
 			systems += system->Name();
+			waypointGovernments.emplace_back(system->GetGovernment());
 		}
 		subs["<waypoints>"] = systems;
+		
+		// Waypoint governments are formatted exactly like stopover governments.
+		subs["<government: waypoints>"] = GovernmentNames(waypointGovernments);
 	}
 	
 	// Instantiate the NPCs. This also fills in the "<npc>" substitution.
 	for(const NPC &npc : npcs)
 		result.npcs.push_back(npc.Instantiate(subs, source, result.destination->GetSystem()));
+	// Fill in the NPC government substitutions, "<government: npc>" and
+	// "<government: npcs>". The singular always refers to the first NPC
+	// block, while the plural refers to all of the mission's NPCs.
+	if(!result.npcs.empty())
+	{
+		if(!result.npcs.front().Ships().empty())
+			subs["<government: npc>"] = result.npcs.front().Ships().front()->GetGovernment()->GetName();
```

Comment:
I think i need to re-check this re: how `<npc>` is filled in. `<npc>` and `<government: npc>` should definitely refer to the same NPC to avoid confusion.